package.json

```diff
@@ -17,24 +17,24 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "mobx": "^6.0.2"
+    "mobx": "^6.3.2"
   },
   "devDependencies": {
-    "@types/jest": "26.0.15",
-    "@typescript-eslint/eslint-plugin": "4.6.1",
-    "@typescript-eslint/parser": "4.6.1",
+    "@types/jest": "26.0.23",
+    "@typescript-eslint/eslint-plugin": "4.26.1",
+    "@typescript-eslint/parser": "4.26.1",
     "benchmark": "2.1.4",
-    "eslint": "7.12.1",
-    "husky": "4.3.0",
-    "jest": "26.6.2",
-    "lint-staged": "10.5.1",
+    "eslint": "7.28.0",
+    "husky": "6.0.0",
```

Comment:
Cool! I think the migration path now requires a `husky.rc` 🤔 

Links:
- [Migration 4 to 6](https://typicode.github.io/husky/#/?id=husky-4-to-6-cli) 
- [Older approach with 6.0 - Section: …but I still want to define hooks in package.json
](https://blog.typicode.com/husky-git-hooks-javascript-config/)

Reply:
🥴